core/src/main/java/org/elasticsearch/index/query/SimpleQueryStringParser.java

```diff
@@ -205,39 +194,33 @@ public Query parse(QueryParseContext parseContext) throws IOException, QueryPars
             field = currentFieldName;
         }
 
-        // Use the default field (_all) if no fields specified
-        if (fieldsAndWeights == null) {
-            field = parseContext.defaultField();
+        SimpleQueryStringBuilder qb = new SimpleQueryStringBuilder(queryBody);
+        qb.boost(boost).fields(fieldsAndWeights).analyzer(analyzerName).queryName(queryName);
+        qb.minimumShouldMatch(minimumShouldMatch).flags(flags);
+        if (defaultOperator != null) {
+            qb.defaultOperator(defaultOperator);
         }
-
-        // Use standard analyzer by default
-        if (analyzer == null) {
-            analyzer = parseContext.mapperService().searchAnalyzer();
+        if (locale != null) {
+            qb.locale(locale);
         }
-
-        if (fieldsAndWeights == null) {
-            fieldsAndWeights = Collections.singletonMap(field, 1.0F);
+        if (lowercaseExpandedTerms != null) {
```

Comment:
Yeah - makes sense. When reading our mail thread I noticed, I would need to retrofit this PR back to one of it's former formats - totally missed that when making the changes...